src/components/FilterBtn/FilterBtn.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { filterTodo } from "../../actions/todoActions";
+
+export const FilterBtn = props => {
```

Comment:
```suggestion
export const FilterBtn = ({ children, filterTodo, active }) => {
```
деструктуризація одразу